yt_dlp/extractor/bibeltv.py

```diff
@@ -1,27 +1,204 @@
 from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    LazyList,
+    clean_html,
+    determine_ext,
+    js_to_json,
+    parse_iso8601,
+    try_get
+)
 
 
 class BibelTVIE(InfoExtractor):
+    IE_DESC = 'BibelTV'
     _VALID_URL = r'https?://(?:www\.)?bibeltv\.de/mediathek/videos/(?:crn/)?(?P<id>\d+)'
+    _RETURN_TYPE = 'video'
```

Comment:
Why did you define this? It should be handled by IE?

Reply:
Could you explain, when it makes sense to use this? Only when returning playlists?

Reply:
See the `InfoExtractor._RETURN_TYPE` property for implementation. If tests are well defined, it shouldnt be necessary to define since it will be calculated

Reply:
Then I remove it.